src/frontend/screens/Settings/ProjectSettings/YourTeam/SelectInviteeRole.tsx

```diff
@@ -0,0 +1,114 @@
+import {StyleSheet, View} from 'react-native';
+import {
+  NativeNavigationComponent,
+  ViewStyleProp,
+} from '../../../../sharedTypes';
+import {defineMessages, useIntl} from 'react-intl';
+import {Text} from '../../../../sharedComponents/Text';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+import {DARK_GREY, LIGHT_GREY} from '../../../../lib/styles';
+import React from 'react';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import {DeviceNameWithIcon} from '../../../../sharedComponents/DeviceNameWithIcon';
+import {RoleWithIcon} from '../../../../sharedComponents/RoleWithIcon';
+
+const m = defineMessages({
+  title: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.SelectInviteeRole.title',
+    defaultMessage: 'Select a Role',
+  },
+  selectingDevice: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.SelectInviteeRole.selectingDevice',
+    defaultMessage: 'You are selecting a role for this device:',
+  },
+  particpantDescription: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.SelectInviteeRole.particpantDescription',
+    defaultMessage:
+      'As a Participant this device can take and share observations. They cannot manage users or project details.',
+  },
+  coordinatorDescription: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.SelectInviteeRole.coordinatorDescription',
+    defaultMessage:
+      'As a Coordinator this device can invite and remove users, and manage project details.',
+  },
+});
+
+export const SelectInviteeRole: NativeNavigationComponent<
+  'SelectInviteeRole'
+> = ({route, navigation}) => {
+  const {formatMessage: t} = useIntl();
+  return (
+    <View style={styles.container}>
+      <Text style={{fontSize: 18, fontWeight: 'bold'}}>
+        {t(m.selectingDevice)}
+      </Text>
+      <DeviceNameWithIcon {...route.params} style={{marginTop: 10}} />
+      <RoleCard
+        style={{marginTop: 20}}
+        role="particpant"
+        onPress={() =>
+          navigation.navigate('ReviewInvitation', {
+            ...route.params,
+            role: 'particpant',
+          })
+        }
+      />
+      <RoleCard
+        style={{marginTop: 10}}
+        role="coordinator"
+        onPress={() =>
+          navigation.navigate('ReviewInvitation', {
+            ...route.params,
+            role: 'coordinator',
```

Comment:
This is used in the `WaitingForInviteAccept` component as the roleId, which is a unique ID. This should be renamed maybe, and use the `roleId`, however I'm not sure if we correctly expose this in the API. I think the API is confusing here, and maybe we can quickly jump on a call to discuss how to improve it.